src/Cache.cpp

```diff
@@ -115,19 +115,19 @@ tr1::shared_ptr<Frame> Cache::GetSmallestFrame()
 	const GenericScopedLock<CriticalSection> lock(*cacheCriticalSection);
 	tr1::shared_ptr<openshot::Frame> f;
 
-	 // Loop through frame numbers
-	 deque<long int>::iterator itr;
-	 long int smallest_frame = -1;
-	 for(itr = frame_numbers.begin(); itr != frame_numbers.end(); ++itr)
-	 {
-		 if (*itr < smallest_frame || smallest_frame == -1)
-			 smallest_frame = *itr;
-	 }
+	// Loop through frame numbers
+	deque<long int>::iterator itr = frame_numbers.begin();
+	long int smallest_frame = *itr;
```

Comment:
If that's a problem here, it's a problem in other places in Cache. Oh, and apologizes for including the Cache changes here. I had the wrong branch checked-out.